macOS/LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/macOS/IPC/DataBrokerProtectionAppToAgentInterface.swift

```diff
@@ -17,48 +17,13 @@
 //
 
 import Foundation
+import DataBrokerProtectionShared
 
 public enum DataBrokerProtectionAppToAgentInterfaceError: Error {
     case loginItemDoesNotHaveNecessaryPermissions
     case appInWrongDirectory
 }
 
-@objc
-public class DataBrokerProtectionAgentErrorCollection: NSObject, NSSecureCoding {
```

Comment:
See https://github.com/duckduckgo/apple-browsers/pull/239#discussion_r1995353864 for what's happened to this